### PR TITLE
Fix localization

### PR DESCRIPTION
**Pull request by henryau53**
When I discovered Planify a few days ago, I thought it's really cool. However, I'm someone with OCD☺, so I fixed a few localization issues. This is my first time making a pull request, so please correct me if I've made any mistakes.
当我前几天发现了 Planify，我就觉得它好酷，不过我有点强迫症，所以我修复了一点点本地化的问题，第一次进行 PR，所以有不对的地方请多指教

### CODE REVIEW

**Comment by Fr4nk1inCs** [comment]
I think it would be better if you add space between Chinese character and Latin punctuation (for new translations after line 1443).

我认为（在 1443 行之后的新翻译中的）汉字和拉丁符号之间加上空格的话会更好。

**Comment by henryau53** [comment]
@Fr4nk1in-USTC
At the beginning, I did do it that way, but it seemed strange, so I removed the spaces. However, I realized that what seems strange to me may not be strange to you. So, I accept your suggestion. Could you please tell me how to modify this PR? Should I close the current PR and submit a new one after making changes?

**Comment by Fr4nk1in-USTC** [comment]
You can just commit to your original branch to modify the PR.

For more information about space in Chinese typography, please refer to [chinese-copywriting-guidelines](https://github.com/sparanoid/chinese-copywriting-guidelines).

**Comment by Fr4nk1inCs** [comment]
> @Fr4nk1in-USTC At the beginning, I did do it that way, but it seemed strange, so I removed the spaces. However, I realized that what seems strange to me may not be strange to you. So, I accept your suggestion. Could you please tell me how to modify this PR? Should I close the current PR and submit a new one after making changes?

@henryau53 

You can just commit to your original branch to modify the PR.

For more information about space in Chinese typography, please refer to [chinese-copywriting-guidelines](https://github.com/sparanoid/chinese-copywriting-guidelines).

**Comment by henryau53** [comment]
> @Fr4nk1in-USTC You can just commit to your original branch to modify the PR.
For more information about space in Chinese typography, please refer to [chinese-copywriting-guidelines](https://github.com/sparanoid/chinese-copywriting-guidelines).

I have known about [chinese-copywriting-guidelines](https://github.com/sparanoid/chinese-copywriting-guidelines) for a long time. This project just provides a good universal template, which is not mandatory to follow as a standard. This kind of universal template is usually suitable for writing documents or code, and I mostly follow this template for my own documentations. However, for some UI development, such as web or client-side, it may not always be applicable. Sometimes, it is difficult to choose between "a good universal template" and what is appropriate.
  
I apologize for the delay in responding to your message. I have been quite busy lately. What do you think if use full-width quotation marks instead of "add space between Chinese character and Latin punctuation" like "添加至“最爱”", I would appreciate your advice on this matter. Big Thanks!

**Comment by Fr4nk1inCs** [comment]
> I have known about [chinese-copywriting-guidelines](https://github.com/sparanoid/chinese-copywriting-guidelines) for a long time. This project just provides a good universal template, which is not mandatory to follow as a standard. This kind of universal template is usually suitable for writing documents or code, and I mostly follow this template for my own documentations. However, for some UI development, such as web or client-side, it may not always be applicable. Sometimes, it is difficult to choose between "a good universal template" and what is appropriate.

I agree. It's hard to find a good universal solution for all problems.

> I apologize for the delay in responding to your message. I have been quite busy lately. What do you think if use full-width quotation marks instead of "add space between Chinese character and Latin punctuation" like "添加至“最爱”", I would appreciate your advice on this matter. Big Thanks!

I think this is the ideal solution. Would you mind modify all the translations? They all use Latin punctuations.

**Comment by henryau53** [comment]
@Fr4nk1in-USTC 

Thank you for your suggestion. It's not a problem, just that the progress may be a bit slower as I've been quite busy lately, and I have also noticed other localization issues that need to be handled.

There is also another question. Some of the modifications in my PR need to be synchronized with the POT file. 

Just like the file `src/Layouts/ProjectRow.vala:412`:

```vala
- favorite_item.title = project.is_favorite ? ("Remove from favorites") : ("Add to favorites");
+ favorite_item.title = project.is_favorite ? _("Remove from favorites") : _("Add to favorites");
```

If the POT file is regenerated, it may affect many other localized files. Who should be responsible for generating the POT file?

File: po/zh_CN.po [hunk 1]
```diff
@@ -885,31 +894,31 @@ msgstr "%Y"
 
 #: src/Widgets/Calendar/CalendarWeek.vala:40
 msgid "Mo"
-msgstr "周一"
```
Comment (henryau53) [inline po/zh_CN.po:888]: Modify it like this so that the calendar doesn't appear too compact.

File: src/Layouts/ItemRow.vala [hunk 2]
```diff
@@ -1571,7 +1571,7 @@ public class Layouts.ItemRow : Gtk.ListBoxRow {
     public void delete_request (bool undo = true) {
         if (board) {
             var dialog = new Adw.MessageDialog ((Gtk.Window) Planner.instance.main_window,
-			                                    _("Delete to-do"), _("Are you sure you want to delete this to-do?"));
```
Comment (henryau53) [inline src/Layouts/ItemRow.vala:1574]: Standardize the “To-Do” word, Better for the application and translation.

**Comment by henryau53** [comment]
I've been quite busy recently, but I managed to find some time and finally released a new version. Please check it out.☺

**Pull request merged**